bug: [PAGOPA-1263] redis exception

alessio-acitelli opened the pull request:
<!--- Please always add a PR description as if nobody knows anything about the context these changes come from. -->
<!--- Even if we are all from our internal team, we may not be on the same page. -->
<!--- Write this PR as you were contributing to a public OSS project, where nobody knows you and you have to earn their trust. -->
<!--- This will improve our projects in the long run! Thanks. -->

#### List of Changes

<!--- Describe your changes in detail -->

- managed redis exception

#### Motivation and Context

<!--- Why is this change required? What problem does it solve? -->

- fixed event loss if redis exception occurs

#### How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, tests ran to see how -->
<!--- your change affects other areas of the code, etc. -->

- Junit test

#### Screenshots (if appropriate):

#### Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

alessio-acitelli committed:
[PAGOPA-1263] redis exception: managed redis exception

alessio-acitelli committed:
[PAGOPA-1263] redis exception: increased int test timeout

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of https://github.com/pagopa/pagopa-negative-biz-events-datastore into PAGOPA-1263-neg-broken-pipe-redis

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of
https://github.com/pagopa/pagopa-negative-biz-events-datastore into
PAGOPA-1263-neg-broken-pipe-redis

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of https://github.com/pagopa/pagopa-negative-biz-events-datastore into PAGOPA-1263-neg-broken-pipe-redis

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of
https://github.com/pagopa/pagopa-negative-biz-events-datastore into
PAGOPA-1263-neg-broken-pipe-redis

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of https://github.com/pagopa/pagopa-negative-biz-events-datastore into PAGOPA-1263-neg-broken-pipe-redis

Commit:
[PAGOPA-1263] neg broken pipe redis: new chart

pagopa-github-bot committed:
Bump to version 0.1.0-3-PAGOPA-1263-neg-broken-pipe-redis [skip ci]

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of https://github.com/pagopa/pagopa-negative-biz-events-datastore into PAGOPA-1263-neg-broken-pipe-redis

pagopa-github-bot committed:
Bump to version 0.1.0-4-PAGOPA-1263-neg-broken-pipe-redis [skip ci]

Commit:
# WARNING: head commit changed in the meantime

Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of
https://github.com/pagopa/pagopa-negative-biz-events-datastore into
PAGOPA-1263-neg-broken-pipe-redis

Commit:
# WARNING: head commit changed in the meantime

Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of
https://github.com/pagopa/pagopa-negative-biz-events-datastore into
PAGOPA-1263-neg-broken-pipe-redis

pagopa-github-bot committed:
Bump to version 0.1.0-5-PAGOPA-1263-neg-broken-pipe-redis [skip ci]

Commit:
Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of https://github.com/pagopa/pagopa-negative-biz-events-datastore into PAGOPA-1263-neg-broken-pipe-redis

pagopa-github-bot committed:
Bump to version 0.1.0-6-PAGOPA-1263-neg-broken-pipe-redis [skip ci]

pasqualespica commented on integration-test/src/features/negative-biz-events-processor.feature:
```diff
@@ -2,36 +2,36 @@ Feature: All about payment events consumed by Azure Function biz-event-processor
   Scenario: An awakable negative biz event published on Event-Hub is skipped by cache
     Given a random "awakable" biz event is published on eventhub
-    When biz event has been properly stored into datastore after 5000 ms
+    When biz event has been properly stored into datastore after 20000 ms
```
why this change ?

pasqualespica replied:
should be housekeeping cosmos db biz

pasqualespica reviewed (approved):
LGTM

pagopa-github-bot committed:
Bump to version 0.2.0 [skip ci]

Commit:
# WARNING: head commit changed in the meantime

Merge branch 'PAGOPA-1263-neg-broken-pipe-redis' of
https://github.com/pagopa/pagopa-negative-biz-events-datastore into
PAGOPA-1263-neg-broken-pipe-redis
